lib/logidze/model.rb

```diff
@@ -50,6 +50,9 @@ def at(ts)
 
       object_at = dup
       object_at.apply_diff(version, log_data.changes_to(version: version))
+      object_at.id = id
```

Comment:
Looks like that was a bug. Can you, please, add specs for this and, maybe, it would be better to extract this fix into a separate PR to release it ASAP?